ada_feeding/config/ada_planning_scene.yaml

```diff
@@ -3,7 +3,7 @@ ada_planning_scene:
   ros__parameters:
     object_ids: # list of objects to add to the planning scene
       - wheelchair
-      - wheelchair_collision
+      # - wheelchair_collision
```

Comment:
TODO: Remove this once we've addressed [the issue with `wheelchair_collision` messing up certain planning calls](https://github.com/personalrobotics/ada_feeding/pull/39#issuecomment-1656435835).

Reply:
Resolved [here](https://github.com/personalrobotics/ada_feeding/pull/39#issuecomment-1661287243)